### ISSUE TITLE
Add evaluation semantics for BGP* and BIND*/FIND into the draft

### ISSUE THREAD

**Issue by hartig** [w3c/rdf-star#8]
Currently, the [Evaluation Semantics section](https://w3c.github.io/rdf-star/#evaluation-semantics) of the draft is just a copy from the corresponding subsection of the [original tech report](http://arxiv.org/pdf/1406.3399). The actual definition of the formal semantics of a BGP* and of the SPARQL* version of BIND is in a different part of that tech report (where it defines ⟦B⟧_G and ⟦(tp AS ?v)⟧_G). These definitions still need to be adapted and moved into the [Evaluation Semantics section](https://w3c.github.io/rdf-star/#evaluation-semantics) of the draft.

**Comment by hartig** [comment]
I have finished writing up the evaluation semantics for BGP* and for BIND*/FIND. The relevant parts of the draft that I have modified in this context are:
1. [Section 4.1 (Initial Definitions)](https://w3c.github.io/rdf-star/rdf-star-cg-spec.html#initial-definitions) has been extended with additional definitions; everything that comes after the green box (NOTE) is new.
2. In [Section 4.4 (Evaluation Semantics)](https://w3c.github.io/rdf-star/rdf-star-cg-spec.html#evaluation-semantics), I have defined the two base cases (point 1 is for BGP* and point 2 is for BIND*/FIND).
